src/pages/en/tutorial/1-setup/2.md

```diff
@@ -0,0 +1,119 @@
+
+7. You may see a message telling you that you have no "staged" commits, and asking you if you want to stage them. Click <kbd>Always</kbd> and continue.
+
+8. Lastly, the list of changed files should be replaced with a <kbd>Publish</kbd> button. Click this to send your committed changes to GitHub.
+
+#### 🌐 Developing in the cloud
+
+As you make changes to your project, you will see a visual indication that some files are no longer in sync with your repository at GitHub.com.
+
+**CodeSandbox**: When there are changes to be committed back to GitHub, the GitHub tab will display a notification. When you first click on the GitHub tab in CodeSandbox, you will be asked to "Link" your sandbox back to your repository. 
+
+After that, clicking on the GitHub tab will list any files with changes that have not been synced back to your GitHub repository. It will allow you to enter a commit message (summary) describing your changes or intention, and an optional description. Make sure that "Commit directly to the main branch" is selected, then press "Commit changes."
+
+**StackBlitz**: When there are changes to be committed back to GitHub, a "Commit" button will appear at the top left of your workspace. Clicking on this will allow you to enter a commit message, describing your changes or intention. Click "Commit" to send these changes back to GitHub and update your repository.
+
+
+#### 🖥️ 🌐 Verify your work
+
+To verify that your project is successfully stored on GitHub, visit GitHub.com again in a browser window and look under your account for a list of your repositories. Choose the new one you created, and verify that it contains your Astro project files.
+
+## Deploy your website on Netlify
+
+Here, you will import your Astro project from GitHub. Netlify will use that project to create a live, public website. 
+
+### Create a new Netlify site
+
+1. Log in to https://app.netlify.com/.
+
+2. Click <kbd>Add new site</kbd> > <kbd>Import an existing project</kbd>.
+
+    You will be asked to connect to a Git provider. Choose GitHub (or your chosen git provider) and, if necessary, follow the steps onscreen to authenticate your GitHub account. Then, choose the new GitHub repository you made to store your `astro-site` project from the list provided.
+
+3. At the final step, Netlify will show you your app's site settings. The defaults should be correct for your Astro project, so you can scroll down and click <kbd>Deploy site</kbd>.
+
+Congratulations, you have an Astro website!
+
+### Make changes locally 
+
+On your Site overview page, you will see your randomly-generated project name, and your website URL in the form `https://project-name-123456.netlify.app`. You can change your project name to something more memorable, and this will automatically update your URL.
+
+### Visit your updated website
+
+Click on the URL in your site settings, or type it in to a browser window to view your new website.
```

Comment:
I think this is needed because this is literally the first time they will visit their site on the web. The point here is not (yet) to see changes, but to see their site exists at all!

I think I'll change the header instead to not say "updated" but more like, YOU HAVE A SITE!  :partying_face: